python/ml4ir/applications/classification/tests/data/configs/feature_config.yaml

```diff
@@ -65,6 +65,10 @@ features:
     log_at_inference: true
     is_group_metric_key: true
     preprocessing_info:
+      - fn: preprocess_text
```

Comment:
This combination of preprocessing effectively perform the tokenization we have with the legacy EP model.